packages/ace-core/src/core/get-all-rules.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const axe = require('axe-core');
+const mustache = require('mustache');
+const fs = require('fs');
+
+const Rules = require('../checker/rules');
+
+const PATH_TO_RULES_TEMPLATE = `${__dirname}/rules.mustache`;
+
+function mapUrlsToKB(axerules) {
+  // TODO: axe helping urls should be map to daisy knowledge database
+  // see kbMap in axe2ace.js
+  return axerules;
+}
+
+function getAllRules() {
```

Comment:
My understanding is that this function returns a static list, not depending on the input, correct?